app/models/metric/statistic.rb

```diff
@@ -1,15 +1,18 @@
 module Metric::Statistic
   def self.calculate_stat_columns(obj, timestamp)
-    return {} unless obj.respond_to?(:container_groups)
-    return {} unless obj.respond_to?(:container_images)
-
+    hash = {}
```

Comment:
FYI `hash` is also a method on `Object`.  I'm not avoiding `hash` in my own code either ;-), it's too useful a word IMHO, but the practical consequence of the clash is that if you access `hash` local var before setting it, it will instead call `hash()` resulting in weird errors about an integer...